### PR TITLE
monero: add livecheck

### PR DESCRIPTION
**Pull request by chenrui333**
monero: add livecheck

---

relates to #74785

### CODE REVIEW

**Comment by nandahkrishna** [comment]
Just adding a note here that they mark the "Latest release" on GitHub (and `0.17.2.0` isn't marked as the latest yet).

File: Formula/monero.rb [hunk 1]
```diff
@@ -7,6 +7,11 @@ class Monero < Formula
   license "BSD-3-Clause"
   revision 1
 
+  livecheck do
+    url "https://www.getmonero.org/blog/tags/releases.html"
+    regex(/Monero GUI .*?v?(\d+(?:\.\d+)+)["' >]/i)
```
Comment (samford) [inline Formula/monero.rb:12]:
```suggestion
    url :stable
    strategy :github_latest
```

**Review by samford — CHANGES_REQUESTED** [review]
The `monero` formula uses a tag from GitHub as `stable`, so it's arguably best to check GitHub for the latest version (to align the check with the `stable` source).

As Nanda mentioned, the GitHub repository marks the newest release as "latest" and the `0.17.2.0` tag isn't the "latest" release yet. Using the `GithubLatest` strategy is appropriate here.

File: Formula/monero.rb (hunk 1 above)
Comment (selsta) [reply Formula/monero.rb:12]: I also would recommend to use the Github stable tag. (I help with monero release engineering).

Comment (chenrui333) [reply Formula/monero.rb]: Sounds good to me.

**Review by samford — APPROVED** [review]

**Pull request merged**